src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java

```diff
@@ -53,40 +43,47 @@ public void setUp() {
 
     @Test
     public void should_not_allow_requests_without_valid_service_authorisation_and_return_403_response_code() {
-
+        Map<String, DmnValue<?>> processVariables = new HashMap<>();
         given()
             .relaxedHTTPSValidation()
             .contentType(APPLICATION_JSON_VALUE)
-            .body(appealSubmittedCreateTaskRequest(caseId)).log().body()
+            .body(new SendMessageRequest("createMessageTask", processVariables)).log().body()
             .baseUri(testUrl)
-            .basePath("/tasks")
+            .basePath("/workflow/message")
             .when()
             .post()
             .then()
             .statusCode(HttpStatus.FORBIDDEN_403);
     }
 
     @Test
-    public void transition_creates_atask_with_default_due_date() {
+    public void transition_creates_a_task_with_default_due_date() {
+
+        Map<String, DmnValue<?>> processVariables = mockProcessVariables(
+            null,
+            "Process Application",
+            "processApplication",
+            "TCW"
+        );
 
         given()
             .relaxedHTTPSValidation()
             .header(SERVICE_AUTHORIZATION, serviceAuthorizationToken)
             .contentType(APPLICATION_JSON_VALUE)
-            .body(appealSubmittedCreateTaskRequest(caseId)).log().body()
+            .body(new SendMessageRequest("createTaskMessage", processVariables)).log().body()
```

Comment:
Minor suggestion for easier mantainance in the future "createTaskMessage" should be an enum as task-managament, see: https://github.com/hmcts/wa-task-management-api/blob/master/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaMessage.java

Reply:
yes, all these tests and logic will move in the next sprint. There is a ticket to move the dueDat to the case event handler. 
I have just put back the original tests and did not spend more than necessary to make it work since as we say it's going to move. Actually, the whole dueDate is subject to major changes, so probably is not worth the effort at this moment.